libs/blocks/list.js

```diff
@@ -7,7 +7,8 @@ const blocks = [
   'fragment',
   'how-to',
   'modal',
-  'review',
+  'ost',
```

Comment:
Not needed unless OST is going to be use on DX Blog or Blog.   This list.js is a legacy file that is only used on those sites.

Reply:
Removed ✅